ports/atmel-samd/mpconfigport.mk

```diff
@@ -65,19 +65,19 @@ CIRCUITPY_SAMD = 1
 endif
 
 ifndef CIRCUITPY_ULAB
-ifneq ($(CIRCUITPY_SMALL_BUILD),1)
+ifneq ($(CIRCUITPY_FULL_BUILD),0)
 CIRCUITPY_ULAB = 1
 endif
 endif
 
 ifndef CIRCUITPY_RGBMATRIX
-ifneq ($(CIRCUITPY_SMALL_BUILD),1)
+ifneq ($(CIRCUITPY_FULL_BUILD),0)
 CIRCUITPY_RGBMATRIX = 1
 endif
 endif
 
 ifndef CIRCUITPY_FRAMEBUFFERIO
-ifneq ($(CIRCUITPY_SMALL_BUILD),1)
+ifneq ($(CIRCUITPY_FULL_BUILD),0)
 CIRCUITPY_FRAMEBUFFERIO = 1
```

Comment:
ditto

Reply:
Ditto for RGBMatrix, this is labeled as though it's a SAMD only feature. is it?

Reply:
Chatted with @jepler about this on Discord to verify that FrameBufferIO is only used with RGBMatrix on the SAMD for now. Both RGBMatrix or FrameBufferIO default to 0 right now (unlike ulab, which defaults on) so they should be defined port by port - I imagine we'll make them default to 1 sometime later once they have more coverage (doesn't seem like we have a specific line for that but I'd figure it's when all the "big" ports atmel-samd, nrf, mimxrt, and stm are supported).